Mathlib/RingTheory/DedekindDomain/IntegralClosure.lean

```diff
@@ -246,9 +247,41 @@ instance integralClosure.isDedekindDomain_fractionRing [IsDedekindDomain A] :
     IsDedekindDomain (integralClosure A L) :=
   integralClosure.isDedekindDomain A (FractionRing A) L
 
+open nonZeroDivisors Algebra IsLocalization IsFractionRing IsScalarTower in
+instance [NoZeroSMulDivisors A C] [Module.Finite A C] :
+    IsLocalization (Algebra.algebraMapSubmonoid C A⁰) (FractionRing C) := by
```

Comment:
This seems to be less general than https://github.com/leanprover-community/mathlib4/pull/22966/files#diff-b07b06029f1c92c14d68e21d04720f1c32a7a33fd174abd1b5e543eba6d17411R590-R599

Reply:
Thanks, I've removed it.